### PR TITLE
MOL-309 `tusd`

### PR DESCRIPTION
**Pull request by omBratteng**
- MOL-309 Update poetry dependencies c6a640c
- MOL-309 Update `README` 10b7b03
- MOL-309 Update `build-upload-url.py` to match correct uploader string 7060dff
- MOL-309 Add function to remove `asyncpg` dialect from psql dsn string abbec13
- MOL-309 Update Dockerfile to use the new structure fa34c4d
- MOL-309 Use the upstream repo for tusd Checks out at specific commit that is known to be working until a release is issued 7010794
- MOL-263 Add `tusd` to dependabot a7b753e
- MOL-309 Remove mailgun from `start.sh` not used ece8c5a
- MOL-309 Add `tusd` to dependabot 4dfcd85
- MOL-309 Fix typo in `README` bcdfb39
- MOL-309 Update test to include dbstring without `+asyncpg` d81b782
- MOL-309 Remove echoing of aws and gcs (not used) 07ea494
- MOL-309 Only do azure tusd params We don't know how well supported the others are, as they're not regularly tested be43558
- MOL-309 Remove another unused env variable 7541394
- MOL-309 Remove `env` defaults for gcs and s3 to not be confusing 5fc2bb2
- MOL-309 Cleans up the output from `Hooks present:` f36bcec
- MOL-309 Update `.env.default` e392948

### CODE REVIEW

File: tusd/start.sh [hunk 1]
```diff
@@ -19,10 +19,6 @@ if [ -n "$AZURE_STORAGE_KEY" ]; then
     echo "AZURE_STORAGE_KEY is set (secret)"
 fi
 
-echo "Mailgun domain: ${MAILGUN_DOMAIN}"
```
Comment (kriwal-arkivverket) [inline tusd/start.sh:22]: Hvorfor har du fjernet dette, men ikke f.eks `AZURE_STORAGE_KEY` på linje 18-20, og `AWS_SECRET_ACCESS_KEY` på linje 11-13?

**Review by kriwal-arkivverket — CHANGES_REQUESTED** [review]

File: tusd/tests/test_hooks_utils.py [hunk 2]
```diff
@@ -68,6 +68,14 @@ def test_get_metadata1(mocker):
     assert data_from_db.ekstern_id == ekstern_id
 
 
+def test_fix_dbstring():
```
[the remark this answers is not in the document]
Comment (omBratteng) [reply tusd/tests/test_hooks_utils.py:71]: Testes i d81b782

File: tusd/start.sh (hunk 1 above)
Comment (omBratteng) [reply tusd/start.sh:22]: Fordi `AZURE_STORAGE_KEY` og `AWS_SECRET_ACCESS_KEY` kan brukes av tusd. Siden vi kan koble tusd mot azure, gcs og s3 om ønsket.
Fjernet `MAILGUN_DOMAIN` fordi det ikke brukes noen andre steder i koden heller, kunne fjernet det herfra også
https://github.com/arkivverket/mottak-k8s-config/blob/master/k8s/templates/tusd/tusd-deployment.yaml#L67-L73

Comment (omBratteng) [reply tusd/start.sh:22]: Også dette?
https://github.com/arkivverket/mottak-k8s-config/blob/master/k8s/templates/tusd/tusd-deployment.yaml#L65-L66

**Review by kriwal-arkivverket — APPROVED** [review]

**Pull request merged**